src/entity/units/goru.ts

```diff
@@ -44,50 +60,328 @@ const unit: UnitSource = {
     death: 'goruDeath'
   },
   init: (unit: Unit.IUnit, underworld: Underworld) => {
+    // TODO - Bug: Undying 2 can be re-added by cloning/splitting/creating a Goru with no undying modifier
+    // Using originalLife might prevent this, but also prevents summoned Goru's from getting undying.
```

Comment:
Is this bug still active? Let's make sure it's tracked in github in addition to this comment.

Reply:
Okay, I opened an issue for it 👍